### DISCUSSION TITLE
2 features I'd love to see: a search bar at the top and a little clock at the bottom or something....

### DISCUSSION THREAD

**Discussion by obrm** [dailydotdev/daily#818] [Feature Requests]
Here's a mock of an optional design:
https://drive.google.com/file/d/1GLAz1IQhm0p0DXIqS2I2rwsx6VlzcVLc/view?usp=share_link

The search can be a Google search or an internal search, or to have a little option dropdown to choose the kind of search, and the user can choose the default search type in the settings...

I know there's a search in the side, but a search bar on the top is more useful and convenient.

**Comment by idoshamun** [comment]
Hi @obrm,

Thanks for the suggestion.
Let me share it internally and get back to you.

**Reply by obrm** [reply]
Hi,
Thank you for replying.
Looking forward to hear from you.

Ori

**Reply by idoshamun** [reply]
@obrm we believe search in the header can be very useful indeed but tbh we think we need to improve it first before showcasing it in the front page. It's not the best search to say the least 😅

**Reply by obrm** [reply]
I had another request - add a small clock somewhere...

**Reply by idoshamun** [reply]
This is out of scope at the moment 🙏

**Reply by obrm** [reply]
Here's a clock component you can use...
https://codesandbox.io/s/useefect-watch-m3re0i
